src/pls/data/nerd_icons.yml

```
@@ -5,31 +5,31 @@ nerd_icons:
   pls:        "\uf444"  # nf-oct-primitive_dot
 
   apple:      "\ue711"  # nf-dev-apple
-  audio:      "\ufc1d"  # nf-mdi-speaker_wireless
-  broom:      "\uf5e1"  # nf-mdi-broom
+  audio:      "\uf1c7"  # nf-fa-file_audio_o
+  broom:      "\ue615"  # nf-seti-config
   config:     "\ue615"  # nf-seti-config
   css:        "\ue749"  # nf-dev-css3
   docker:     "\uf308"  # nf-linux-docker
   folder:     "\uf07b"  # nf-fa-folder
-  git:        "\uf7a1"  # nf-mdi-git
+  git:        "\uf47f"  # nf-oct-git_compare
   github:     "\uf408"  # nf-oct-mark_github
   html:       "\ue736"  # nf-dev-html5
-  image:      "\uf7e8"  # nf-mdi-image
+  image:      "\uf1c5"  # nf-fa-file_image_o
```

Only removing the page flap from the top-right corner.
```suggestion
  image:      "\uf03e"  # nf-fa-image
```